File: Server/NewsTwitter/Model/news.cs
using System;
namespace WoeMobile.Model
{
	/// <summary>
	/// news:实体类(属性说明自动提取数据库字段的描述信息)
	/// </summary>
	[Serializable]
	public partial class news
	{
		public news()
		{}
		#region Model
		private int _idnews;
		private string _intro;
		private DateTime? _gen_time;
		private int? _writer;
        private manager manager;
        private int verify_id;
        private String username;
        private String userid;

        public String Userid
        {
            get { return userid; }
            set { userid = value; }
        }

        public String Username
        {
            get { return username; }
            set { username = value; }
        }

        public int Verify_id
        {
            get { return verify_id; }
            set { verify_id = value; }
        }
        private DateTime verify_date = DateTime.MinValue;

        public DateTime Verify_date
        {
            get { return verify_date; }
            set { verify_date = value; }
        }
        private int verify;

        public int Verify
        {
            get { return verify; }
            set { verify = value; }
        }
        public manager Manager
        {
            get { return manager; }
            set { manager = value; }
        }
		private string _title;
		private string _content;
		private int? _itemid;
        private item item;

        public item Item
        {
            get { return item; }
            set { item = value; }
        }

		private int? _passcheked;
		private String _keyword;
		private int? _default_news;
		private int? _clicked;
		private string _news_url;
		private string _image_url;
		private int? _comment_sum;
		/// <summary>
		/// auto_increment
		/// </summary>
		public int idnews
		{
			set{ _idnews=value;}
			get{return _idnews;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string intro
		{
			set{ _intro=value;}
			get{return _intro;}
		}
		/// <summary>
		/// 
		/// </summary>
		public DateTime? gen_time
		{
			set{ _gen_time=value;}
			get{return _gen_time;}
		}
		/// <summary>
		/// 
		/// </summary>
		public int? writer
		{
			set{ _writer=value;}
			get{return _writer;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string title
		{
			set{ _title=value;}
			get{return _title;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string content
		{
			set{ _content=value;}
			get{return _content;}
		}
		/// <summary>
		/// 
		/// </summary>
		public int? itemid
		{
			set{ _itemid=value;}
			get{return _itemid;}
		}
		/// <summary>
		/// 
		/// </summary>
		public int? passcheked
		{
			set{ _passcheked=value;}
			get{return _passcheked;}
		}
		/// <summary>
		/// 
		/// </summary>
		public String keyword
		{
			set{ _keyword=value;}
			get{return _keyword;}
		}
		/// <summary>
		/// 
		/// </summary>
		public int? default_news
		{
			set{ _default_news=value;}
			get{return _default_news;}
		}
		/// <summary>
		/// 
		/// </summary>
		public int? clicked
		{
			set{ _clicked=value;}
			get{return _clicked;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string news_url
		{
			set{ _news_url=value;}
			get{return _news_url;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string image_url
		{
			set{ _image_url=value;}
			get{return _image_url;}
		}
		/// <summary>
		/// 
		/// </summary>
		public int? comment_sum
		{
			set{ _comment_sum=value;}
			get{return _comment_sum;}
		}
		#endregion Model

	}
}

File: Android/NewsClient/src/cn/com/dc/app/client/bean/News.java
package cn.com.dc.app.client.bean;

import java.util.Date;

import android.os.Parcel;
import android.os.Parcelable;

public class News implements Parcelable {
	public News()
	{
		
	}
	public int getIdnews() {
		return idnews;
	}
	public void setIdnews(int idnews) {
		this.idnews = idnews;
	}
	public String getIntro() {
		return intro;
	}
	public void setIntro(String intro) {
		this.intro = intro;
	}
	public Date getGentime() {
		return gentime;
	}
	public void setGentime(Date gentime) {
		this.gentime = gentime;
	}
	public int getWriter() {
		return writer;
	}
	public void setWriter(int writer) {
		this.writer = writer;
	}
	public int getVerifyid() {
		return verifyid;
	}
	public void setVerifyid(int verifyid) {
		this.verifyid = verifyid;
	}
	public String getUsername() {
		return username;
	}
	public void setUsername(String username) {
		this.username = username;
	}
	public String getUserid() {
		return userid;
	}
	public void setUserid(String userid) {
		this.userid = userid;
	}
	public String getTitle() {
		return title;
	}
	public void setTitle(String title) {
		this.title = title;
	}
	public String getContent() {
		return content;
	}
	public void setContent(String content) {
		this.content = content;
	}
	public int getItemid() {
		return itemid;
	}
	public void setItemid(int itemid) {
		this.itemid = itemid;
	}
	public int getPasscheked() {
		return passcheked;
	}
	public void setPasscheked(int passcheked) {
		this.passcheked = passcheked;
	}
	public String getKeyword() {
		return keyword;
	}
	public void setKeyword(String keyword) {
		this.keyword = keyword;
	}
	public int getDefaultnews() {
		return defaultnews;
	}
	public void setDefaultnews(int defaultnews) {
		this.defaultnews = defaultnews;
	}
	public int getClicked() {
		return clicked;
	}
	public void setClicked(int clicked) {
		this.clicked = clicked;
	}
	public String getNewsurl() {
		return newsurl;
	}
	public void setNewsurl(String newsurl) {
		this.newsurl = newsurl;
	}
	public String getImageurl() {
		return imageurl;
	}
	public void setImageurl(String imageurl) {
		this.imageurl = imageurl;
	}
	public int getCommentsum() {
		return commentsum;
	}
	public void setCommentsum(int commentsum) {
		this.commentsum = commentsum;
	}
	private String newsurl;
	private String imageurl;
	private int commentsum = 0;
	@Override
	public int describeContents() {
		// TODO
		return 0;
	}
	@Override
	public void writeToParcel(Parcel dest, int flags) {
		// TODO
		dest.writeString(newsurl);
		dest.writeString(imageurl);
		dest.writeInt(commentsum);
		
		dest.writeInt(idnews);
		dest.writeInt(writer);
		dest.writeInt(verifyid);
		dest.writeInt(itemid);
		
		dest.writeInt(defaultnews);
		dest.writeInt(clicked);
		dest.writeString(keyword);
		dest.writeString(content);
		
		dest.writeString(title);
		dest.writeString(userid);
		dest.writeString(username);
		
		dest.writeValue(gentime);
	}
	private int idnews;
	private String intro;
	private Date gentime;
	private int writer;

	private int verifyid;
	private String username;
	private String userid;

	private String title;
	private String content;
	private int itemid;

	private int passcheked;
	private String keyword;
	private int defaultnews;
	private int clicked;
	public static final Parcelable.Creator<News> CREATOR = new Parcelable.Creator<News>() {
		public News createFromParcel(Parcel in) {
			return new News(in);
		}

		public News[] newArray(int size) {
			return new News[size];
		}
	};

	private News(Parcel in) {
		newsurl = in.readString();
		imageurl= in.readString();
		commentsum= in.readInt();
		
		idnews= in.readInt();
		writer= in.readInt();
		verifyid= in.readInt();
		itemid= in.readInt();
		
		defaultnews= in.readInt();
		clicked= in.readInt();
		keyword= in.readString();
		content= in.readString();
		
		title= in.readString();
		userid= in.readString();
		username= in.readString();
		
		gentime = (Date) in.readValue(Date.class.getClassLoader());
	}
	
	
}
